integration/init.go

```diff
@@ -44,6 +51,18 @@ func init() {
 	if clusterID == "" {
 		os.Setenv(EnvVarClusterID, ClusterID())
 	}
+
+	var err error
+	versionBundleVersion, err = framework.GetVersionBundleVersion(service.NewVersionBundles(), TestedVersion())
+	if err != nil {
+		panic(err.Error())
+	}
+	// TODO there should be a not found error returned by the framework in such
+	// cases.
+	if VersionBundleVersion() == "" {
+		panic("version bundle version  must not be empty")
+	}
+	os.Setenv(EnvVarVersionBundleVersion, VersionBundleVersion())
```

Comment:
Is setting this value here really needed?

Reply:
The tests require it. The version bundle version is used in the `AWSConfig` CR. This is done in each test.